modules/virt-editing-vmis-web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/cnv-manage-vmis.adoc
+
+[id="virt-editing-vmis-web_{context}"]
+= Editing a standalone virtual machine instance using the web console
+
+You can edit text and annotations for a standalone virtual machine instance using the web console. Other items displayed in the *Overview* page for a standalone virtual machine instance are not editable.
```

Comment:
`text` should be `labels`. 
`Overview` should be `Details`, `overview` is generally for review, `details` page is used to for VM settings, which is supposing editable for VM, but not VMI.

Reply:
Made the changes. Please review.